console/module/data/src/main/java/org/eclipse/kapua/app/console/module/data/client/AssetTabItem.java

```diff
@@ -177,6 +181,30 @@ public void componentSelected(ButtonEvent ce) {
         resultsTableTabItem.add(resultsTable);
         resultsTabPanel.add(resultsTableTabItem);
 
+        Window.addResizeHandler(new ResizeHandler() {
```

Comment:
This seems to be the same code of the following three classes in this PR.
The only thing that changes it seems to be the "limit value". 
I strongly recommend create a `ConsoleResizeHandler(BorderLayoutData messageLayout, int widthLimit)` used by all of the classes in order to avoid where possible duplicated code.